app/src/main-process/main.ts

```diff
@@ -406,19 +406,6 @@ app.on(
 function createWindow() {
   const window = new AppWindow()
 
-  if (__DEV__) {
```

Comment:
This existed only to install the plugins, since they are being removed, this can go away as well.